modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -25,25 +25,33 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 For security, Red Hat recommends first setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
 
-. To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section. For example:
-.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
-+
-|===
-| *Key* | *Value*
-|Name
-|administrator
-|Role
-|Admin
-|===
-.. If you are using the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/authentication_and_authorization/index#configuring-htpasswd-identity-provider[HTPasswd Identity Provider] with the username `UserA` that is part of the group `GroupA`, you can use the following key-value pairs to create access rules:
-+
-|===
-| *Key* | *Value*
-|Name
-|UserA
-|Group
-|GroupA
-|UserID
-|<UUID>
-|===
-. Click *Save*.
+. Optional: To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section, then enter the rule information and click *Save*. You will need attributes for the user or group so that you can configure access. 
++
+[TIP]
+====
+Group mappings are more robust because groups are usually associated with teams or permissions sets and require modification less often than users.
+====
++
+
+To get user information in {ocp}, you can use one of the following methods:
++
+--
+* Use the link:https://docs.openshift.com/container-platform/4.11/rest_api/user_and_group_apis/user-user-openshift-io-v1.html[{ocp} API].
```

Comment:
> @kcarmichael08 Just a quick question - Are the ACS docs ONLY on docs.openshift.com? If so, then I'm fine with this hard link to d.o.c. If not, and it's on the portal too, then this hard link could be problematic. Thanks!

They are also on the portal, but in a different branch than the OCP docs, so xrefs don't work, even in assemblies. Should I link to the portal version instead?

Reply:
I think if anything this is a really rare case - I know that usually we don't have links to RH docs in modules, and especially hard links - so I will take out if not allowed.

Reply:
Ohhh, I also forgot - sorry, this was kind of late Friday when I was working on this - there is no portal version of the OCP API docs (too big), so that's why I hard linked the d.o.c. version. (I had forgotten why.) But, if the rule is no link at all, I can remove it.

Reply:
There's not a good answer to this one. IMO, unless using the API is the most preferred method, I suggest either removing it altogether or removing the link and moving it to the bottom of the list. (In most cases, it's acceptable to document the one best or preferred option.)

Reply:
Oh, I did forget that the API docs were still only on d.o.c. @kalexand-rh since this is the case, do you think this is acceptable to keep then (if the answer to your question above is that @kcarmichael08 wants to keep both options)?

Reply:
@rukletsov Hi Alex, is the API preferred method of getting user info? If not, would it be OK to move the API method to the bottom of the list of three methods (gui/yaml, k8s yaml, API), and remove the link to the API doc? There are limitations to what kind of links our documentation allows to other RH documentation.

Reply:
I think it would be acceptable.

Reply:
Acceptable, yes.